Client/Frontend/Settings/SyncContentSettingsViewController.swift

```diff
@@ -178,6 +181,17 @@ class SyncContentSettingsViewController: SettingsTableViewController, FeatureFla
         )
     }
 
+    private func loginsSyncEnabledTelemetry(status: Bool) {
+        TelemetryWrapper.recordEvent(
+            category: .action,
+            method: .tap,
+            object: .loginsSyncEnabled,
+            extras: [
+                TelemetryWrapper.ExtraKey.isCreditCardSyncToggleEnabled.rawValue: status
```

Comment:
This extra doesn't have the right key. It should be `EventExtraKey.isLoginSyncEnabled.rawValue`